provider/pkg/await/deployment.go

```diff
@@ -689,17 +687,15 @@ func (dia *deploymentInitAwaiter) checkReplicaSetStatus() {
 // This is used to determine whether the deployment is rolling out a new revision, which in turn, creates/updates a
 // replica set.
 func (dia *deploymentInitAwaiter) changeTriggeredRollout() bool {
-	if dia.config.lastInputs == nil {
+	if dia.config.lastOutputs == nil {
```

Comment:
This change is to address this feedback:
https://github.com/pulumi/pulumi-kubernetes/pull/2756/files#r1473847833